frontend/src/actionCreators/staticContentActionCreator.js

```diff
@@ -11,144 +10,140 @@ import { createRequestHeader } from "@/utils/RequestHeaders";
 
 export const fetchMineDisturbanceOptions = () => (dispatch) => {
   dispatch(request(reducerTypes.GET_DISTURBANCE_OPTIONS));
-  dispatch(showLoading("modal"));
```

Comment:
Removed the loading state on all StaticContent actionCreators as they are being fetched when a parent loads, and not in a modal, therefore this loading state is never seen.